src/opnsense/mvc/app/controllers/OPNsense/OpenVPN/forms/dialogInstance.xml

```diff
@@ -432,6 +432,28 @@ Set to 0 to disable, remember to change your client as well.
             <visible>false</visible>
         </grid_view>
     </field>
+    <field>
+        <id>instance.auth-gen-token-renewal</id>
+        <label>Auth Token Renewal</label>
+        <type>text</type>
+        <advanced>true</advanced>
+        <style>role role_server</style>
+        <help>How often the auth token will be renewed, token expire after 2 * renewal time.</help>
+        <grid_view>
+            <visible>false</visible>
+        </grid_view>
+    </field>
+    <field>
+        <id>instance.auth-gen-token-secret</id>
```

Comment:
@mhalden If I understand the option correctly, these should be generated via `openvpn --genkey auth-token`, in which case it't probably practical if we add a button there (and maybe validate the input as well as this seems to be pretty strict).

Reply:
Yup, that's correct. It would definitely be useful with a button to generate the secret, but for our case we're fine without since it's basically set it and forget it.

Reply:
@mhalden ok, thanks, let me fiddle a bit with this then and merge the batch.